src/tensorflow.cc

```diff
@@ -1332,62 +1342,196 @@ AutoCompleteHelper::FixBatchingSupport()
   return nullptr;  // success
 }
 
+bool
+AutoCompleteHelper::IsEmptyIO(triton::common::TritonJson::Value& io_config) {
```

Comment:
Instead of here, I believe it make more sense to add the Empty function in triton_json.h itself.
It can be useful across the board.
The function call then in L1363 would become: io_config.Empty()

Reply:
I agree with putting this in the TritonJson api but is calling it Empty() a standards thing? I have always preferred the `IsEmpty()` myself since the function returns a boolean and Empty() sounds like it could be clearing the value or returning an empty one. I've had to conform both ways and only have a weak opinion on this.

Reply:
Updated the corresponding Common repository PR with `Empty()`  and now use that change in tensorflow backend

Reply:
IsEmpty sounds better! I was just saying that you can add this in TritonJson.

Reply:
Changed to IsEmpty()